test/numerical/TestMatMul2D.cpp

```diff
@@ -41,8 +40,10 @@ int main(int argc, char *argv[]) {
   setCompilerOption(OptionKind::CompilerOptLevel, "3");
   llvm::cl::ParseCommandLineOptions(
       argc, argv, "TestMatMul2D\n", nullptr, "TEST_ARGS");
-  bool success;
+  std::cout << "Target options: \""
```

Comment:
Not a comment on this line.  
What about the other numerical tests? Can you also print the Target options for them pls.

Reply:
sure